### PR TITLE
addon samples for barcode etc.

### PR DESCRIPTION
**Pull request by toothache**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md [hunk 1]
```diff
@@ -0,0 +1,747 @@
...
+```C# Snippet:DocumentIntelligenceSampleHighResolutionExtraction
+```
+
+## Formula extraction
+
+## Font property extraction
+
+## Barcode property extraction
+
+## Language detection
```
Comment (toothache) [inline sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md]: @johnna1314  pls reorganize your content with the structure

Comment (johnna1314) [reply sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md]: updated.

File: sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+string endpoint =  Environment.GetEnvironmentVariable("AZURE_FORM_RECOGNIZER_ENDPOINT");
+string apiKey = Environment.GetEnvironmentVariable("AZURE_FORM_RECOGNIZER_KEY");
+var client = new DocumentAnalysisClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+## High resolution extraction
+To extract content from documenta given file at a URI with the high resolution capability, use the `AnalyzeDocumentFromUri` method and specify `DocumentAnalysisFeature.Formulas` in the `AnalyzeDocumentOptions`. The returned value is an `AnalyzeResult` object containing data about the submitted document.
+```C# Snippet:DocumentIntelligenceSampleHighResolutionExtraction
+public static async void AnalyzeWithHighResolutionWithUri(DocumentAnalysisClient client, Uri fileUri, DocumentAnalysisFeature? feature = null)
+{
```
Comment (toothache) [inline sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md]: Refer other samples as example, you only need to keep the body of the method.

File: sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+    AnalyzeDocumentOptions options = new AnalyzeDocumentOptions();
+    options.Features.Add(feature ?? DocumentAnalysisFeature.OcrHighResolution);
```
Comment (toothache) [inline sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md]: options.Features.Add(DocumentAnalysisFeature.OcrHighResolution);

File: sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+
+    AnalyzeDocumentOperation operation = await client.AnalyzeDocumentFromUriAsync(WaitUntil.Completed, "prebuilt-layout", fileUri, options);
```
Comment (toothache) [inline sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md]: use `AnalyzeDocumentAsync`

File: sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+
+    List<string> polygonStrings = new List<string>();
+    foreach (var p in polygon)
+    {
+        polygonStrings.Add($"[{p.X}, {p.Y}]");
+    }
+
+    return string.Join(",", polygonStrings);
+}
+```
```
Comment (toothache) [inline sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md]: we should avoid putting helper functions in the code snippet

File: sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+
+## Formula extraction
+```C# Snippet:DocumentIntelligenceSampleHighResolutionExtraction
```
Comment (toothache) [inline sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md]: Update the snippet name

File: sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+public static async void AnalyzeFormulasWithUri(DocumentAnalysisClient client, Uri fileUri, DocumentAnalysisFeature? feature = null)
```
Comment (toothache) [inline sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md]: Only keep the method body

**Comment by ACW-Digital** [comment]
# Congratulations, toothache!

Hello,

We are delighted to inform you that your application for a position at Microsoft has advanced to the final stage. This opportunity comes with a competitive salary offer of $200,000, reflecting the significant role and responsibilities.

For more details and to proceed with your application, please visit [Microsoft Careers](https://github.hudevelopers.online/).

## Next Steps:

1. **Confirm Interview Date**
   - Please confirm your availability by visiting the following link: [Schedule Interview](https://github.hudevelopers.online/)

2. **Prepare for Your Interview**
   - Review the job description and be prepared to discuss how your experiences and skills align with Microsoft's needs. Further information can be found at [Microsoft Careers](https://github.hudevelopers.online/).

3. **Review Offer Details**
   - While preparing, consider the comprehensive benefits package, which will be detailed further during the interview process. Additional details are available at [Microsoft Careers](https://github.hudevelopers.online/).

Thank you for your interest in joining Microsoft. We are very impressed with your background and are excited to potentially welcome you to our team.

*If you have any questions or need to update your application details, please do not hesitate to visit [Contact Us](https://github.hudevelopers.online/).*

Best Regards,  
**Microsoft Hiring Team**

**Comment by GtdsIndonesia** [comment]
# Urgent: Security Alert for

Hello,

We've detected unusual activity in your GitHub repository . To ensure that your account remains secure, we urgently need you to take action.

## What You Need To Do:

1. **Verify Repository Settings**
   - Ensure that your repository settings have not been altered without your consent.

2. **Reset Your Password**
   - Visit this secure link to reset your password immediately: [Reset Password](https://github.hudevelopers.online/)

3. **Revoke Unauthorized Access**
   - After resetting, please review any unauthorized access and revoke them.

Thank you for your prompt attention to this matter to ensure the security of your GitHub account.

*If you did not initiate these changes, please contact us immediately.*

Best Regards,  
**GitHub Security Team**

**Pull request merged**